cba_settings_userconfig/cba_settings.sqf

```diff
@@ -714,13 +714,13 @@ force force emr_main_staminaCoefficient = 1;
 force force emr_main_whitelistStr = "";
 
 // F/A-18
-js_jc_fa18_advancedStart = false;
-js_jc_fa18_atflirRequire = true;
-js_jc_fa18_canopyLoss = false;
+force force js_jc_fa18_advancedStart = true;
+force force js_jc_fa18_atflirRequire = true;
+force force js_jc_fa18_canopyLoss = true;
 js_jc_fa18_cursorSensitivity = 1;
 js_jc_fa18_interactCursor = false;
-js_jc_fa18_interactionRadiusMod = 1;
-js_jc_fa18_mav_tdcDepressRequiredForMove = false;
+js_jc_fa18_interactionRadiusMod = 1.10;
```

Comment:
nah, it's just a convenience setting recommended by Churizo but players should be able to change it in case it interacts weirdly for other monitor resolutions.